prime-router/metadata/schemas/covid-19.schema

```diff
@@ -841,6 +842,7 @@ elements:
     hl7Field: SPM-10
     referenceUrl: https://hl7-definition.caristix.com/v2/HL7v2.5.1/Fields/SPM.10
 
+  # should this also have a lookup into LIVD?
   - name: specimen_description
```

Comment:
We could add it to LIVD, but I don't think that the description is dependent on the type of device used like other fields are.